game.py

```diff
@@ -1,35 +1,18 @@
-import datetime
-import json
-import os
-import random
 import time
-import typing
-from itertools import chain
 
 import pyxel
 
-from board import Board
-from calculator import clamp, attack, get_setl_totals, complete_construction, attack_setl
-from catalogue import get_available_improvements, get_available_blessings, get_available_unit_plans, get_heathen, \
-    get_default_unit, get_improvement, get_blessing, get_unit_plan, Namer, FACTION_COLOURS, PROJECTS, get_project
-from menu import Menu, MainMenuOption, SetupOption, WikiOption
-from models import Player, Settlement, Construction, OngoingBlessing, CompletedConstruction, Unit, HarvestStatus, \
-    EconomicStatus, Heathen, AttackPlaystyle, GameConfig, Biome, Victory, VictoryType, AIPlaystyle, \
-    ExpansionPlaystyle, UnitPlan, OverlayType, Faction, ConstructionMenu, Project
-from movemaker import MoveMaker, set_player_construction
-from music_player import MusicPlayer
-from overlay import SettlementAttackType, PauseOption
-from save_encoder import SaveEncoder, ObjectConverter
-
-# The prefix attached to save files created by the autosave feature.
-AUTOSAVE_PREFIX = "auto"
-# The directory where save files are created and loaded from.
-SAVES_DIR = "saves"
+from game_management.game_controller import GameController
+from game_management.game_input_handler import on_key_arrow_down, on_key_arrow_up, on_key_arrow_left,\
+    on_key_arrow_right, on_key_return, on_mouse_button_right, on_mouse_button_left, on_key_shift, on_key_c, on_key_f, \
+    on_key_d, on_key_tab, on_key_space, on_key_m, on_key_s, on_key_n, on_key_b, on_key_escape, on_key_a
+from game_management.game_state import GameState
 
 
 class Game:
```

Comment:
Conflicts to resolve in here :(

Reply:
Righto, I ported that J key event to the game_input_handler and the OverlayType.HEAL to the overlay check for the ESC handler, but a sanity check would be worthwhile